core/src/com/po/fuck/model/Entity.java

```diff
@@ -50,21 +54,33 @@ public boolean takeDamage(float damage) {
         return true;
     }
 
-    public float getHP(){
+    public float getHP() {
         return healthPoints;
     }
 
-    public Managed<Movement> getMovement(){
+    public Managed<Movement> getMovement() {
         return movement;
     }
 
-    public Managed<Weapon> getWeapon(){
+    public Managed<Weapon> getWeapon() {
         return weapon;
     }
 
     @Override
     public void destructor() {
+        System.out.println("Entity destroyed");
```

Comment:
Remove debug output!

Reply:
Sorry, I started search with "System.out", for some reason it did not show me that.